setup.py

```diff
@@ -26,31 +26,30 @@ def read(file_name):
 def getPkgPath():
     return __import__(PACKAGE).__path__[0] + '/'
 
-
 def minify(files, outfile, ftype):
-    import requests
+    """ using jsmin and cssmin, minify the list of javascript/css files, and write it in the outfile """
+    from jsmin import jsmin
+    from cssmin import cssmin
     import io
+    # make a dictionary to associate file types with the tool/function to minify it
+    tools = {
+        'js': jsmin,
+        'css': cssmin
+    }
 
+    # put all content from files in one variable
+    # minified files contents are separated by a newline
     content = ''
-
     for filename in files:
         with io.open(getPkgPath() + filename, 'r', encoding='utf8') as f:
-            content = content + '\n' + f.read()
-
-    data = {
-        'code': content,
-        'type': ftype,
-    }
-    response = requests.post('http://api.applegrew.com/minify', data)
-    response.raise_for_status()
-    response = response.json()
-    if response['success']:
-        with io.open(getPkgPath() + outfile, 'w', encoding='utf8') as f:
-            f.write(response['compiled_code'])
-    else:
-        raise Exception('%(error_code)s: "%(error)s"' % response)
-
-
+            content = content + '\n' + tools[ftype](f.read())
```

Comment:
Careful, these minifiers might throw errors. You should check what errors they might raise and handle some of them.
The string concatenation used here is not very efficient. As a rule of thumb prefer something like:
```python
'\n'.join(iterable)
# over
mystring = mystring + part 
```